src/Speckle.Sdk/Transports/MemoryTransport.cs

```diff
@@ -54,7 +54,7 @@ public object Clone()
 
   public string TransportName { get; set; } = "Memory";
 
-  public Action<ProgressArgs>? OnProgressAction { get; set; }
+  public IProgress<ProgressArgs>? OnProgressAction { get; set; }
```

Comment:
This is no longer an action, let's open a ticket to align naming after we merge this an deal with it's consequences :)